### PR TITLE
Add docu for highlighing of correction round

### PR DESCRIPTION
**Pull request by nicolasruscher**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, editor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.

### Motivation and Context

Currently the documentation for the highlighting of the second correction round is missing. 
### Description

This PR adds a short description of the feature to the existing documentation

### Steps for Testing

1. Log in to Artemis
2. Navigate to Course Administration
3. ...

### CODE REVIEW

**Review by lschlesinger — COMMENTED** [review]
A suggestion: "This feature is currently _available_..."

**Review by MDK5 — CHANGES_REQUESTED** [review]
I don't understand "batch" - do you mean badge?

File: docs/user/exams/instructors_guide.rst [hunk 1]
```diff
@@ -307,6 +307,7 @@ If you want you can also enable the :ref:`second correction <Before the exam's a
 
 - To assess a submission a second time go to the exercise assessment dashboard. When it is enabled, a |start_new_assessment| button will be visible in the second correction round.
 - The new second assessment will have all the feedback copied from the first assessment. Those can be overridden, and new feedback can be added as well. This does not override the original result, but saves a separate second result.
+- Within the second correction round assessment instructors and tutors can highlight which feedback was created for which correction round. This is displayed as a batch at the bottom of every feedback. This view can be enabled or disabled any time during a second correction round assessment. The feature is currently possible for text, modeling and fileupload exercises.
```
Comment (FuchsDominik) [inline docs/user/exams/instructors_guide.rst:310]:
```suggestion
- Within the second correction instructors and tutors can highlight which feedback was created for which correction round. This is displayed as a batch at the bottom of every feedback. This view can be enabled or disabled any time during a second correction round assessment. The feature is currently possible for text, modeling and file-upload exercises.
```

**Review by FuchsDominik — COMMENTED** [review]
Stumbled across the first sentence. The _assessment_ seems a bit doppelt gemoppelt to me

File: docs/user/exams/instructors_guide.rst (hunk 1 above)
Comment (MaisiKoleni) [inline docs/user/exams/instructors_guide.rst:310]:
```suggestion
- Within the second correction round assessment instructors and tutors can highlight which feedback was created for which correction round. This is displayed as a badge at the bottom of every feedback. This view can be enabled or disabled any time during the second correction by/with.... The feature is currently possible for text, modeling and fileupload exercises.
```
Also see suggestion from Dominik. I would add especially how users can toggle this feature.

**Review by MaisiKoleni — CHANGES_REQUESTED** [review]
See review comment.

File: docs/user/exams/instructors_guide.rst (hunk 1 above)
Comment (nicolasruscher) [reply docs/user/exams/instructors_guide.rst]: done

Comment (nicolasruscher) [reply docs/user/exams/instructors_guide.rst]: done

**Comment by nicolasruscher** [comment]
> I don't understand "batch" - do you mean badge?

I corrected it

**Review by MDK5 — APPROVED** [review]
Looks good to me

**Review by coolchock — DISMISSED** [review]
Text looks good :+1:

**Review by jpbernius — APPROVED** [review]

**Review by ivohashamov — APPROVED** [review]

**Pull request merged**